modules/environment-roles/templates/terraform_restore_db_policy.json.tpl

```diff
@@ -0,0 +1,67 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Action": [
+        "rds:RestoreDBClusterToPointInTime",
```

Comment:
Minor thing, but could you sort these alphabetically? It makes it easier to review a list of permissions.

Reply:
I've sorted these, and the resources below :+1: